lib/cs_guide/resources/venue.ex

```diff
@@ -34,7 +35,7 @@ defmodule CsGuide.Resources.Venue do
   @doc false
   def changeset(venue, attrs \\ %{}) do
     venue
-    |> cast(attrs, [:venue_name, :postcode, :phone_number])
+    |> cast(attrs, [:venue_name, :postcode, :phone_number, :cs_score])
```

Comment:
**TODO** (_on post launch iteration_): **`phone_number`** should be refactored to use `Fields.phone_number` https://github.com/dwyl/fields/issues/17 which will do pre-storage validation. (_which we appear to not have right now, so people could submit "blah!" or `<script>` as the venue phone number and the App would accept it blindly ..._)